FFXIVClientStructs/FFXIV/Component/GUI/AtkUnitBase.cs

```diff
@@ -21,16 +21,86 @@ public unsafe partial struct AtkUnitBase : ICreatable {
     [FieldOffset(0x160)] public AtkValue* AtkValues;
-    [FieldOffset(0x182)] public byte Flags;
-    [FieldOffset(0x189)] public byte UnkFlags189;
+    [FieldOffset(0x168)] public StdVector<Pointer<byte>> CachedAtkValueStrings; // set here: "48 8D 54 24 ?? E8 ?? ?? ?? ?? 48 83 C4 20 41 5E"
+
+    /// <summary>
+    /// <code>
+    /// DepthLayer:<br/>
+    ///   Getter: (Flags180 >> 16) &amp; 0xF<br/>
+    ///   Mask: 0b0000_0000_0000_1111_0000_0000_0000_0000<br/>
+    /// <br/>
+    /// Visibility(?) Flags:<br/>
+    ///   Getter: (Flags180 >> 20) &amp; 0xF<br/>
+    ///   Mask: 0b0000_0000_1111_0000_0000_0000_0000_0000<br/>
+    ///   Values:
+    ///     0b0010 = Is visible<br/>
+    ///     0b0100 = Is hidden due to modal (like Retainer Menu)<br/>
+    /// <br/>
+    /// Applied Visibility(?) Flags:<br/>
+    ///   Getter: (Flags180 >> 24) &amp; 0xF<br/>
+    ///   Mask: 0b0000_1111_0000_0000_0000_0000_0000_0000<br/>
+    ///   Values: same as above<br/>
+    /// <br/>
+    /// UldLoadState:<br/>
+    ///   Getter: (Flags180 >> 28) &amp; 0xF<br/>
+    ///   Mask: 0b1111_0000_0000_0000_0000_0000_0000_0000<br/>
+    ///   Values:
+    ///     0 = Not loaded<br/>
+    ///     1 = UldResource loaded<br/>
+    ///     2 = UldManager finished loading the uld
+    /// </code>
+    /// </summary>
+    [FieldOffset(0x180)] public uint Flags180;
+
+    /// <summary>
+    /// <code>
+    /// 0b1000_0000 = Disable auto-focus (not adding it to Focused Units list)
+    /// </code>
+    /// </summary>
+    [FieldOffset(0x188)] public byte Flags188;
+
+    /// <summary>
+    /// <code>
+    /// 0b0000_0001 = OnSetup was called (= IsReady)
+    /// </code>
+    /// </summary>
+    [FieldOffset(0x189)] public byte Flags189;
+    [FieldOffset(0x18A)] public byte Flags18A;
+
+    /// <summary>
+    /// <code>
+    /// 0b0100_0000 = Don't show on open
+    /// </code>
+    /// </summary>
+    [FieldOffset(0x18D)] public byte Flags18D;
+
     [FieldOffset(0x194)] public uint OpenTransitionDuration;
     [FieldOffset(0x198)] public uint CloseTransitionDuration;
+
     [FieldOffset(0x1A1)] public byte NumOpenPopups; // used for dialogs and context menus to block inputs via ShouldIgnoreInputs
+
     [FieldOffset(0x1A4)] public float OpenTransitionScale;
     [FieldOffset(0x1A8)] public float CloseTransitionScale;
     [FieldOffset(0x1AC)] public float Scale;
+
+    /// <summary>
+    /// An optional scd resource that is loaded along with the uld resource in <see cref="LoadUldResourceHandle"/>.<br/>
+    /// Mainly used by Gold Saucer addons. Handled in AtkModule handler 50.<br/>
+    /// The following scds can be loaded:
+    /// <code>
+    /// 0 = sound/system/SE_UI.scd
+    /// 1 = sound/system/SE_GS.scd
+    /// 2 = sound/system/SE_TTriad.scd
+    /// 3 = sound/system/SE_EMJ.scd
+    /// 4 = sound/system/SE_10thMG.scd
+    /// </code>
+    /// </summary>
+    [FieldOffset(0x1B4)] public byte ScdResourceIndex;
```

Comment:
The remark on this doesn't fully make sense unless SE_UI is part of the not fully loaded base

Reply:
Removed it.